packages/devextreme/js/__internal/ui/text_box/m_text_editor.mask.ts

```diff
@@ -48,23 +54,41 @@ const buildInMaskRules = {
   },
 };
 
-function isNumericChar(char) {
+function isNumericChar(char): boolean {
   return /[0-9]/.test(char);
 }
 
-function isLiteralChar(char) {
+function isLiteralChar(char): boolean {
   const code = char.charCodeAt();
   return code > 64 && code < 91 || code > 96 && code < 123 || code > 127;
 }
 
-function isSpaceChar(char) {
+function isSpaceChar(char): boolean {
   return char === ' ';
 }
 
-const TextEditorMask = TextEditorBase.inherit({
+class TextEditorMask<
+  TProperties extends TextEditorBaseProperties= TextEditorBaseProperties,
+> extends TextEditorBase<TProperties> {
+  _changedValue?: any;
+
+  _maskStrategy!: MaskStrategy;
+
+  _$hiddenElement!: dxElementWrapper;
+
+  _typingDirection?: 'forward' | 'backward';
```

Comment:
It is possible to create a separate type

Reply:
These changes do not aim to make TypeScript typing great for each places. All improvements within your area will be handled by the squad in step 2. I sure that the squad understands the code in its own area better.